### PR TITLE
Template update win mac

### PR DESCRIPTION
**Pull request by PhBrz**

### CODE REVIEW

File: config-dev/Security inventory/macOS/Examples/Template.zsh [hunk 1]
```diff
@@ -1,18 +1,38 @@
...
+#
+#     This allows you to handle dependencies, such as only getting the SSID and encryption method when WIFI is enabled and connected.
+#     If you separated these values into separate tests (WIFI enabled, current SSID, encryption method), it would be more difficult to calculate the risk score and merge the data.
+#
+#     Example: vlGroupSimilarValues.
+#
+# 2)  Splunk has a default limit of 10,000 characters for a single event; data is truncated if it exceeds this limit.
+#     If you expect a result to exceed this limit, consider breaking it into smaller, more manageable pieces.
+#
+# 3)  The Securty Score Splunk dashboard currently does not support every json structure.
```
Comment (vl-federico) [inline config-dev/Security inventory/macOS/Examples/Template.zsh:18]: Typo: Securty -> Security

**Review by vl-federico — APPROVED** [review]
Only one minor comment.

File: config-dev/Security inventory/Windows/Examples/Template.ps1 [hunk 2]
```diff
@@ -0,0 +1,497 @@
+# This file should help you to get started with writing SCI tests for Windows.
+
+### Requires elevated privileges? Then add the following line to the top of the script
+# #Requires -RunAsAdministrator
+
+### We support PowerShell 3.0 and later, so we need to add the following line to the top of the script
+#Requires -Version 3.0
+
+# Include helper functions, like retrun handling, error handling, some registry magic etc.
```
Comment (DominikBritz) [inline config-dev/Security inventory/Windows/Examples/Template.ps1:9]:
```suggestion
# Include helper functions like return handling, error handling, and for working with the registry more easily
```

File: config-dev/Security inventory/Windows/Examples/Template.ps1 [hunk 3]
```diff
@@ -0,0 +1,497 @@
...
+#
+# 3)  The Security Score Splunk dashboard currently does not support every json structure.
+#
+#     Example: Get-vlSupportMatrixExample.
+#
+
+function Get-vlSimpleExample() {
+   <#
+   .SYNOPSIS
+       This test returns a simple result.
```
Comment (DominikBritz) [inline config-dev/Security inventory/Windows/Examples/Template.ps1:34]: Indentation is not consistent

**Review by DominikBritz — APPROVED** [review]

**Pull request merged**